test/csuite/scope/main.c

```diff
@@ -161,7 +161,8 @@ cursor_scope_ops(WT_SESSION *session, const char *uri)
 
         /* Remove any leftover key/value pair, start fresh. */
         SET_KEY;
-        testutil_check(cursor->remove(cursor));
+        if (cursor->search(cursor) == 0)
+            testutil_check(cursor->remove(cursor));
```

Comment:
In general, I'd avoid this semantic because there's a race between `search` and `remove`, it's safer, at least in general, to handle the potential error return where the record isn't there.

Reply:
I am going add another function `testutil_check_error_ok` that does something similar to `testutil_check` but allows for certain errors (in this case `WT_NOTFOUND`) as I don't think we should be failing the test if we try to remove something that does not exist. Let me know if you have anything else/better in mind.